delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/EventsModule.scala

```diff
@@ -32,13 +33,15 @@ object EventsModule extends ModuleDef {
         xas: Transactors,
         jo: JsonKeyOrdering
     ) =>
-      SseEventLog(
-        sseEncoders,
-        organizations.fetch(_).void,
-        projects.fetch(_).map { p => (p.value.organizationUuid, p.value.uuid) },
-        config.sse,
-        xas
-      )(jo)
+      toCatsIO(
```

Comment:
A lot of the things that we inject from the modules are resolver resolution or contexts from files in the classpath.
With my change, they are now loaded with a Cats IO and instead of changing them, I prefered to migrate the modules themselves as it is a more useful change.
So the modules that have not migrated yet, get these kind of temporary changes but we won't have to go back to those that have been migrated